core/server/worker/src/test/java/alluxio/worker/page/PagedBlockStoreCommitBlockTest.java

```diff
@@ -0,0 +1,278 @@
+    dirConf.set(PropertyKey.WORKER_PAGE_STORE_DIRS, ImmutableList.of(dirPath));
+    dirConf.set(PropertyKey.WORKER_PAGE_STORE_SIZES, ImmutableList.of(Constants.MB));
+    dirConf.set(PropertyKey.WORKER_PAGE_STORE_TYPE, PageStoreType.LOCAL);
+    PageStoreDir pageStoreDir =
+            PageStoreDir.createPageStoreDir(
+                    new CacheEvictorOptions().setEvictorClass(FIFOCacheEvictor.class),
+                    PageStoreOptions.createForWorkerPageStore(dirConf).get(DIR_INDEX));
+
+    try  {
+      mUfs = new NoopUfsManager();
+      mConf = Configuration.global();
+      cacheManagerConf.set(PropertyKey.WORKER_PAGE_STORE_PAGE_SIZE, mPageSize);
+      cacheManagerConf.set(PropertyKey.WORKER_PAGE_STORE_DIRS, ImmutableList.of(dirPath));
+
+      // Here mock BlockMasterClientPool and BlockMasterClient since I have no idea
+      // about how to override them.
+      // mockedPool will return a mocked BlockMasterClient when require() is called,
+      // and do nothing when releasing, maybe add some action later on.
+      mBlockMasterClientPool = mock(BlockMasterClientPool.class);
+      mMockedBlockMasterClient = mock(BlockMasterClient.class);
+      when(mBlockMasterClientPool.acquire()).thenReturn(mMockedBlockMasterClient);
+      doNothing().when(mBlockMasterClientPool).release(any());
+      mWorkerId = new AtomicReference<>(-1L);
+      mCacheManagerOptions = CacheManagerOptions.createForWorker(cacheManagerConf);
+      pageStoreDirs = new ArrayList<PageStoreDir>();
+      pageStoreDirs.add(pageStoreDir);
+      mDirs = PagedBlockStoreDir.fromPageStoreDirs(pageStoreDirs);
+    } catch (Exception e) {
+      throw e;
+    }
+
+    mListener = mock(BlockStoreEventListener.class);
+    doAnswer((i) -> {
+      assertEquals(BLOCK_ID, i.getArguments()[0]);
+      return 0;
+    }).when(mListener).onCommitBlockToLocal(anyLong(), any(BlockStoreLocation.class));
+    doAnswer((i) -> {
+      assertEquals(BLOCK_ID, i.getArguments()[0]);
+      return 0;
+    }).when(mListener).onCommitBlockToMaster(anyLong(), any(BlockStoreLocation.class));
+  }
+
+  // This Test case success both to commit, no Exception should be thrown,
+  // and both onCommit method should be called
+  @Test
+  public void localCommitAndMasterCommitBothSuccess()
+         throws IOException, InterruptedException, TimeoutException {
+    try {
+      mPageMetaStore = new PagedBlockMetaStore(mDirs) {
+        // here commit always success
+        @Override
+        public PagedBlockMeta commit(long BLOCK_ID) {
+            return super.commit(BLOCK_ID);
+        }
+      };
+      mCacheManager = CacheManager.Factory.create(mConf, mCacheManagerOptions, mPageMetaStore);
+
+      mPagedBlockStore = new PagedBlockStore(mCacheManager, mUfs, mBlockMasterClientPool, mWorkerId,
+              mPageMetaStore, mCacheManagerOptions.getPageSize());
+    } catch (IOException e) {
+      throw new RuntimeException();
+    }
```

Comment:
no need to wrap the exception since the test method throws IOException in the declaration